plugin-sdk/che-plugin-sdk-ext-plugins/src/main/java/org/eclipse/che/ide/ext/tutorials/client/wizard/ExtensionPagePresenter.java

```diff
@@ -196,10 +196,12 @@ private void validateCoordinates() {
     private String getAttribute(String attrId) {
         Map<String, List<String>> attributes = dataObject.getProject().getAttributes();
         List<String> values = attributes.get(attrId);
-        if (!(values == null || values.isEmpty())) {
-            return values.get(0);
+        if (values == null || values.isEmpty()) {
+            return "";
         }
-        return "";
+
+        String value = values.get(0);
+        return value != null ? value : "";
```

Comment:
the same [comment](https://github.com/codenvy/che-plugins/pull/303/files#r41338746)